README.md

```diff
@@ -0,0 +1,121 @@
+Autopilot Pattern Docker Registry
+=================================
+
+*A private Docker registry with automatic SSL support implemented according to the [Autopilot Pattern](http://autopilotpattern.io/) for automatic discovery and configuration.*
+
+[![DockerPulls](https://img.shields.io/docker/pulls/autopilotpattern/docker-registry.svg)](https://registry.hub.docker.com/u/autopilotpattern/docker-registry/)
+[![DockerStars](https://img.shields.io/docker/stars/autopilotpattern/docker-registry.svg)](https://registry.hub.docker.com/u/autopilotpattern/docker-registry/)
+
+### Hello world example
+
+1. [Get a Joyent account](https://my.joyent.com/landing/signup/) and [add your SSH key](https://docs.joyent.com/public-cloud/getting-started).
+1. Install the [Docker Toolbox](https://docs.docker.com/installation/mac/) (including `docker` and `docker-compose`) on your laptop or other environment, as well as the [Joyent Triton CLI](https://docs.joyent.com/public-cloud/api-access/cloudapi).
```

Comment:
`Install [Docker](https://www.docker.com/products/docker) (includes `docker-compose` and `docker-machine`)`